machineconfiguration/v1/types_pinnedimageset.go

```diff
@@ -0,0 +1,105 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=pinnedimagesets,scope=Cluster
+// +kubebuilder:subresource:status
+// +openshift:api-approved.openshift.io=https://github.com/openshift/api/pull/2198
+// +openshift:file-pattern=cvoRunLevel=0000_80,operatorName=machine-config,operatorOrdering=01
+// +openshift:enable:FeatureGate=PinnedImages
+// +kubebuilder:metadata:labels=openshift.io/operator-managed=
+
+// PinnedImageSet describes a set of images that should be pinned by CRI-O and
+// pulled to the nodes which are members of the declared MachineConfigPools.
+//
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
+// +openshift:compatibility-gen:level=1
+type PinnedImageSet struct {
+	metav1.TypeMeta `json:",inline"`
+	
+	// metadata is the standard object metadata.
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec describes the configuration of this pinned image set.
+	// +required
+	Spec PinnedImageSetSpec `json:"spec"`
+
+	// status describes the last observed state of this pinned image set.
+	// +optional
+	Status PinnedImageSetStatus `json:"status,omitempty"`
+}
+
+// PinnedImageSetStatus describes the current state of a PinnedImageSet.
+type PinnedImageSetStatus struct {
+	// conditions represent the observations of a pinned image set's current state.
+	// +patchMergeKey=type
+	// +patchStrategy=merge
+	// +kubebuilder:validation:MaxItems=10
```

Comment:
doing a `make update` locally causes this to regen, so I think this might be missing from the generated CRs

Reply:
resolved